IAC-Test/deploy_userdata.sh

```
@@ -0,0 +1,72 @@
+sudo apt update
+
+# Install Java (OpenJDK 11)
+echo "Installing Java..."
+sudo apt install -y openjdk-11-jdk
+
+# Check Java installation
+java -version
+
+# Set JAVA_HOME environment variable
+echo "Setting JAVA_HOME environment variable..."
+JAVA_HOME=$(dirname $(dirname $(readlink -f $(which java))))
```

Shellcheck

[warning] Quote this to prevent word splitting.

(SC2046)